modules/ROOT/pages/amazon/amazon-dynamodb-connector-reference.adoc

```diff
@@ -1,1190 +1,1167 @@
+| Target Value a| String |  An expression to evaluate against the operation's output and store the expression outcome in the target variable |  `#[payload]` | 
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors. |  | 
 |===
 
-=== Output
-
+==== Output
 [%autowidth.spread]
 |===
-| Type a| DeleteItemResult
-| Attributes Type a| RequestIDAttribute
+|Type |<<DeleteItemResult>>
+| Attributes Type a| <<RequestIDAttribute>>
 |===
 
 === For Configurations
-
-* config
-
-=== Throws
-
-
-* DYNAMODB:ConditionalCheckFailedException
-* DYNAMODB:CONNECTIVITY
-* DYNAMODB:InternalServerError
-* DYNAMODB:ItemCollectionSizeLimitExceededException
-* DYNAMODB:LimitExceededException
-* DYNAMODB:ProvisionedThroughputExceededException
-* DYNAMODB:ResourceInUseException
-* DYNAMODB:ResourceNotFoundException
-* DYNAMODB:RETRY_EXHAUSTED
-* DYNAMODB:Unknown
+* <<config>> 
+
+==== Throws
+* DYNAMODB:ACCESS_DENIED_EXCEPTION 
+* DYNAMODB:CONDITIONAL_CHECK_FAILED_EXCEPTION 
+* DYNAMODB:CONNECTIVITY 
+* DYNAMODB:INCOMPLETE_SIGNATURE_EXCEPTION 
+* DYNAMODB:ITEM_COLLECTION_SIZE_LIMIT_EXCEEDED_EXCEPTION 
+* DYNAMODB:LIMIT_EXCEEDED_EXCEPTION 
+* DYNAMODB:MISSING_AUTHENTICATION_TOKEN_EXCEPTION 
+* DYNAMODB:PROVISIONED_THROUGHPUT_EXCEEDED_EXCEPTION 
+* DYNAMODB:RESOURCE_IN_USE_EXCEPTION 
+* DYNAMODB:RESOURCE_NOT_FOUND_EXCEPTION 
+* DYNAMODB:RETRY_EXHAUSTED 
+* DYNAMODB:THROTTLING_EXCEPTION 
+* DYNAMODB:UNKNOWN 
+* DYNAMODB:UNRECOGNIZED_CLIENT_EXCEPTION 
+* DYNAMODB:VALIDATION_EXCEPTION 
 
 
 [[deleteTable]]
-== Delete Table
-
+=== Delete Table
 `<dynamodb:delete-table>`
 
 
-The DeleteTable operation deletes a table and all of its items.
+ The DeleteTable operation deletes a table and all of its items. See the Amazon http://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_DeleteTable.html[API Reference] for this operation. 
 
 
-=== Parameters
-
-[%header%autowidth.spread]
+==== Parameters
+[%header,cols="20s,20a,30a,20a,10a"]
 |===
 | Name | Type | Description | Default Value | Required
-| Configuration | String | The name of the configuration to use. | |x
-| Table Name a| String |  The name of the table to delete. |  |x
-| Target Variable a| String |  The name of a variable where the output of this operation is stored |  |
-| Target Value a| String |  An expression to evaluate against the operation's output and the outcome of that expression is stored in the target variable. |  `#[payload]` |
-| Reconnection Strategy a| * reconnect
-* reconnect-forever |  A retry strategy in case of connectivity errors |  |
+| Configuration | String | The name of the configuration to use. | | x
+| Table Name a| String |  The name of the table to delete. |  | x
+| Target Variable a| String |  The name of a variable to store the operation's output. |  | 
+| Target Value a| String |  An expression to evaluate against the operation's output and store the expression outcome in the target variable |  `#[payload]` | 
```

Comment:
An expression to evaluate against the operation's output. The evaluation outcome is stored in the target variable.